tough/src/schema/spki.rs

```diff
@@ -125,8 +125,7 @@ fn asn1_encode_len(n: usize) -> Vec<u8> {
 /// Encode an object identifier in ASN.1.
 #[allow(clippy::cast_possible_truncation)]
 fn asn1_encode_oid(oid: &[u64]) -> Vec<u8> {
-    let mut v = Vec::new();
-    v.push((oid.get(0).unwrap_or(&0) * 40 + oid.get(1).unwrap_or(&0)) as u8);
+    let mut v = vec![((oid.get(0).unwrap_or(&0) * 40 + oid.get(1).unwrap_or(&0)) as u8)];
```

Comment:
```suggestion
    let mut v = vec![(oid.get(0).unwrap_or(&0) * 40 + oid.get(1).unwrap_or(&0)) as u8];
```
I have to admit this one confused me at first!  I don't think about single-element tuples much, so I thought this was creating one, but without a trailing comma it's just a parenthetical expression.  Rust (rustc/rustfmt/clippy) apparently allows one set of extra parens, but any more are trimmed down to one (still extra) set, probably to allow for the programmer to clarify expressions as needed but not go overboard.  In any case, this one isn't needed, and I think it's clearer to avoid it so the reader doesn't make unnecessary associations with tuples.